swift_browser_ui_frontend/src/components/ObjectDeleteButton.vue

```diff
@@ -56,22 +56,17 @@ export default {
         this.$store.state.db.objects.bulkDelete(objIDs);
       }
       swiftDeleteObjects(
+        this.$route.params.project,
         this.$route.params.container,
         to_remove,
-      ).then(() => {
-        if(this.$route.name === "SharedObjects") {
-          this.$store.dispatch(
-            "updateSharedObjects", 
```

Comment:
I suspect that this one should have stayed. We only need to fetch objects again in the SharedObjects view, as it doesn't use the IDB, and now it is not refreshing the shared objects.